contracts/Pool/PoolRegistry.sol

```diff
@@ -320,7 +319,7 @@ contract PoolRegistry is Ownable2StepUpgradeable {
     }
 
     /**
-     * @param comptroller The Comptroller implementation address.
+     * @param comptroller The comptroller proxy address associated to the pool.
      * @notice Returns Venus pool.
```

Comment:
use @return tag for return values?

Reply:
Fixed , can verify the changes at [commit](https://github.com/VenusProtocol/isolated-pools/pull/151/commits/ffd493f85fd85b0e1ede68a5b5b8ca0e298698e4).